windows-agent/internal/daemon/testutils/testutils.go

```diff
@@ -0,0 +1,79 @@
+// Package testutils exports test helpers to be used in other packages that need to change internal behaviors of the daemon.
+package testutils
+
+import (
+	"fmt"
+	"os"
+	"slices"
+	"strings"
+	"syscall"
+	"testing"
+
+	"github.com/canonical/ubuntu-pro-for-wsl/common/testdetection"
+)
+
+// MockWslSystemCmd mocks commands running inside the WSL system distro.
+// Add it to your package_test with:
+//
+//	func TestWithWslSystemMock(t *testing.T) { daemontests.MockWslSystemCmd(t) }
```

Comment:
That description will change when we explicit enable it instead of running `init()`.

Reply:
Not sure, as though I replaced init() properly, I couldn't find a way to have that test case in other package tests, like in the `agent_test`, that allows replacing the `wsl.exe --system` subprocess without doing this.

Reply:
Now I got what you meant.